Update README.md to include protocgen example (#21)
confluentinc/ccloud-sdk-go-v1-public: README.md

> Public Golang SDK for Confluent Cloud.

## To generate / update `*pb.go` files

> * Install `protoc-gen-go`, 

Reviewer: Can you write out the full command?

> * run `go mod vendor` in `ccloud-sdk-go-v1-public`

Reviewer: I think it's already assumed we're in this repo.
```suggestion
* Run `go mod vendor`
```

> * at `ccloud-sdk-go-v1-public` root dir, run the following command:

Reviewer: Make a note to swap out `org/org.proto` with the relevant file?

## To generate / update `*pb.go` files

* Install `protoc-gen-go`:
```
go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

* Run `go mod vendor`

> * At `ccloud-sdk-go-v1-public` root dir, run the following command, replace `org/org.go` with relevant file

Reviewer: I think the command above also implies that we're in the root directory, so it should be fine to leave this out.
```suggestion
* Run the following command, replacing `org/org.go` with the relevant file
```

```
protoc --go_out=. --go_out=/ org/org.proto -I=./ -I=vendor/github.com/gogo/protobuf \
  -I=vendor/github.com/envoyproxy/protoc-gen-validate \
  -I=vendor 
```

> * Remove vendor folder and make a PR

Reviewer: ```suggestion
* Remove the `vendor/` directory and open a PR
```
